LICENSE.md

```diff
@@ -1 +1,295 @@
+  - **Serbia**: EuroGeoGraphics
+  - **South Africa**: Revolutionary GIS
+  - **South Africa**: South Africa Municipal Demarcation Board (SAMDB)
+  - **Taiwan**: National Land Surveying and Mapping Center
+  - **Tanzania**: Tanzania National Bureau of Statistics (TNBS)
+  - **Togo**: ISCGM
+  - **Togo**: Direction General of Cartography - DGC Togo
+  - **Ukraine**: Revolutionary GIS
+  - **Ukraine**: OCHA Ukraine ДНВП Картографія
+  - **Zimbabwe**: Zimbabwe Central Statistics Office (ZimStat)
+  - **Zimbabwe**: OCHA HDX
+- **Metazen**: source [data](https://www.mapzen.com/) and [license](https://github.com/whosonfirst-data/whosonfirst-data/blob/master/LICENSE.md#license)
+- **Mini Tenders**: source [data](http://www.thebolditalic.com/articles/1101-mini-tenders) and [license](http://www.thebolditalic.com/articles/1101-mini-tenders)
+- **Natural Earth**: source [data](http://www.naturalearthdata.com/) and [license](http://www.naturalearthdata.com/about/terms-of-use/)
+- **Geonorge Norway**: source [data](https://www.geonorge.no/) and [license](https://kartkatalog.geonorge.no/metadata/kartverket/administrative-enheter-kommuner/041f1e6e-bdbc-4091-b48f-8a5990f3cc5b)
+- **City of New Orleans, Office of Information Technology and Innovation, Enterprise Information Team**: source [data](https://data.nola.gov/Geographic-Base-Layers/Neighborhoods/92zg-wzkq) and [license](https://data.nola.gov/Geographic-Base-Layers/Neighborhoods/92zg-wzkq)
+- **Oakland Community and Economic Development Department**: source [data](https://data.oaklandnet.com/Property/Oakland-Neighborhoods/7zky-kcq9) and [license](https://data.oaklandnet.com/Property/Oakland-Neighborhoods/7zky-kcq9/about)
+- **City of Oulu**: source [data](http://www.ouka.fi/oulu/oulu-tietoa/avoin-data-aineisto) and [license](http://www.ouka.fi/oulu/oulu-tietoa/kayttoehdot)
+- **OurAirports**: source [data](http://ourairports.com/data/) and [license](http://ourairports.com/)
+- **Pediacities**: source [data](http://catalog.opendata.city/dataset/pediacities-nyc-neighborhoods) and [license](http://catalog.opendata.city/dataset/pediacities-nyc-neighborhoods/resource/91778048-3c58-449c-a3f9-365ed203e914)
+- **Office of Geodesy and Cartography, Poland**: source [data](http://www.gugik.gov.pl/pzgik/dane-bez-oplat/dane-z-panstwowego-rejestru-granic-i-powierzchni-jednostek-podzialow-terytorialnych-kraju-prg) and [license](http://www.gugik.gov.pl/prawa-autorskie)
+- **City of Portland Bureau of Planning and Sustainability**: source [data](http://gis.pdx.opendata.arcgis.com/datasets/c11815647b3949faa20b16cf50ab214d_125) and [license](https://www.arcgis.com/home/item.html?id=c11815647b3949faa20b16cf50ab214d)
+- **Directorate-General for the Territorial Development of Portugal**: source [data](http://www.dgterritorio.pt/dados_abertos/caop/) and license, 
+- **Quattroshapes**: source [data](http://www.quattroshapes.com/) and [license](https://github.com/foursquare/quattroshapes/blob/master/LICENSE.md)
```

Comment:
Same as Mesoshapes in extra sourcing language...

Reply:
fixed in 5827205